### DISCUSSION TITLE
Contribution graph colors shade

### DISCUSSION THREAD

**Discussion by HebaMak** [community/community#51010] [New to GitHub]
Hello everyone,
is there a way to restore the contribution graph shade colors?
now I have most of my contributions become in light green and I need to make over 45 contributions to have the darker green.
if I make a litter number daily, does it help restore the previous colors?

**Comment by pie972** [comment]
GitHub calculates the shades of the contribution graph based on the number of contributions you make in a day. If you make more contributions in a day, the shade of the contribution will be darker, and if you make fewer contributions, the shade will be lighter.
So, if you consistently make contributions every day, even if it's a small number, it will eventually add up and help restore the previous colors of your contribution graph.
